sys/net/gnrc/routing/simple_subnets/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = gnrc_ipv6_simple_subnets
```

Comment:
Module location should match the module name, so

` sys/net/gnrc/routing/simple_subnets/` ⇒ ` sys/net/gnrc/routing/ipv6_simple_subnets/`?